asym_crypto_yaml/crypto.py

```diff
@@ -0,0 +1,148 @@
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import serialization
+from cryptography.hazmat.primitives.asymmetric import padding, rsa
+from cryptography.hazmat.primitives import hashes
+import yaml
+import base64
+
+# See https://cryptography.io/en/latest/
+
+def encode(s):
+    return base64.encodebytes(s).decode('ascii')
+
+def decode(b):
+    return base64.decodebytes(b.encode('ascii'))
+
+
+class Encrypted(str):
```

Comment:
I've always been wary of subclassing built-in types, since operations on them will likely not produce the derived class.  For example, `Encrypted("").lower()` will not produce an Encrypted, but a str.  I don't understand all the ways these are being used, so I guess it's OK? Maybe a docstring here warning of the risks?